src/game.cpp

```diff
@@ -6920,8 +6921,22 @@ static void zones_manager_shortcuts( const catacurses::window &w_info, faction_i
     shortcut_print( w_info, point( tmpx, 4 ), c_white, c_light_green, _( "<Enter>-Edit" ) );
 
     tmpx = 1;
+
+    std::string all_zones = "all";
```

Comment:
Please do no split `Show all` etc into two words because other languages do not necessarily put words in that order or use exactly two words.

Reply:
@Qrox should be good to go, I have left the strings intact